website/docs/r/project_config.html.markdown

```diff
@@ -66,7 +66,7 @@ Nested schema for **definition**:
 	* `name` - (Required, String) The configuration name. It is unique within the account across projects and regions.
 	  * Constraints: The maximum length is `128` characters. The minimum length is `1` character. The value must match regular expression `/^[a-zA-Z0-9][a-zA-Z0-9-_ ]*$/`.
 	* `resource_crns` - (Optional, List) The CRNs of resources associated with this configuration.
-	  * Constraints: The list items must match regular expression `/(?!\\s)(?!.*\\s$)^(crn)[^'"`<>{}\\s\\x00-\\x1F]*/`. The maximum length is `110` items. The minimum length is `0` items.
+	  * Constraints: The list items must match regular expression `/(?!\\s)(?!.*\\s$)^(crn)[^'"<>{}\\s\\x00-\\x1F]*/`. The maximum length is `110` items. The minimum length is `0` items.
```

Comment:
this is not rendered well

<img width="1039" alt="Screenshot 2024-03-12 at 10 46 17" src="https://github.com/IBM-Cloud/terraform-provider-ibm/assets/27769809/a9bbe784-8289-429e-b17c-f9e2a83e1a60">